src/App.js

```diff
@@ -34,7 +35,10 @@ class App extends Component {
           <Route exact path="/flexicon/InternalContactOverview" component={ InternalContactOverview } />
           <Route exact path="/flexicon/orders" component={ OrdersPage }/>
           <Route exact path="/flexicon/customers" component={ CustomerList } />
+          <Route exact path="/flexicon/orders/:orderNumber" component={ OrderDetails }/>
+          <Route exact path="/flexicon/companies" component={ InternalContactOverview }/>
           <Route exact path="/flexicon/orders/:orderId" component={ OrderDetails }/>
```

Comment:
... right here

Reply:
means, the first one is not needed and should be deleted

Reply:
I will delete then :smile:

Reply:
:+1: